doc/source/whatsnew/v1.5.0.rst

```diff
@@ -1236,6 +1236,7 @@ Metadata
 ^^^^^^^^
 - Fixed metadata propagation in :meth:`DataFrame.melt` (:issue:`28283`)
 - Fixed metadata propagation in :meth:`DataFrame.explode` (:issue:`28283`)
+- Fixed metadata propagation for binary operators on :class:`DataFrame` (:issue:`28283`)
```

Comment:
Could you move this to `v1.6.0.rst`?

Reply:
Moved it to `v1.6.0.rst`. Wasn't exactly sure where to put it, please let me know if it's ok.